internal/manifest/iso_tree.go

```diff
@@ -67,8 +59,8 @@ func NewISOTree(m *Manifest,
 func (p *ISOTree) getOSTreeCommits() []osTreeCommit {
 	return []osTreeCommit{
 		{
-			checksum: p.ostree.Parent,
-			url:      p.ostree.URL,
+			checksum: p.OSTree.Checksum,
+			url:      p.OSTree.URL,
 		},
```

Comment:
`p.OSTree` is now pointer and you may end up dereferencing `nil`.

Reply:
Good catch.  Bunch of these slipped by it seems.